stemcell_builder/stages/bosh_ntpd/assets/ntp.conf

```diff
@@ -0,0 +1,13 @@
+driftfile /var/lib/ntp/ntp.drift
+
+statistics loopstats peerstats clockstats
+filegen loopstats file loopstats type day enable
+filegen peerstats file peerstats type day enable
+filegen clockstats file clockstats type day enable
+
+restrict -4 default kod notrap nomodify nopeer noquery
+restrict -6 default kod notrap nomodify nopeer noquery
+
+restrict 127.0.0.1
```

Comment:
@cppforlife , the line `restrict 127.0.0.1` is used to allow unrestricted access from the localhost so that bosh-agent can query localhost by using `ntpq` to get time and offset.